dev-python/pyside/pyside-9999.ebuild

```diff
@@ -1,71 +1,93 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
 
-PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
 
 inherit cmake-utils flag-o-matic python-r1 virtualx git-r3
 
 DESCRIPTION="Python bindings for the Qt framework"
-HOMEPAGE="https://wiki.qt.io/Pyside"
+HOMEPAGE="https://wiki.qt.io/PySide2"
 EGIT_REPO_URI=(
-	"git://code.qt.io/pyside/${PN}.git"
-	"https://code.qt.io/git/pyside/${PN}.git"
+	"git://code.qt.io/pyside/pyside.git"
+	"https://code.qt.io/git/pyside/pyside.git"
 )
 
 LICENSE="LGPL-2.1"
-SLOT="0"
+SLOT="2"
 KEYWORDS=""
 
-IUSE="X declarative designer help multimedia opengl script scripttools sql svg test webkit xmlpatterns"
+IUSE="
+	concurrent declarative designer gui help multimedia network opengl
+	printsupport script scripttools sql svg test testlib webchannel webengine
+	webkit websockets widgets x11extras xmlpatterns"
+
+# The requirements below were strongly inspired by their "PyQt5" equivalents.
```

Comment:
Sorry, my comment was a little cryptic I guess, let me explain.

I do agree that maintaining USE flag parity with PyQt5 (at least as much as possible) *is* a worthwhile goal. What I meant to say here is simply that the internal inter-dependencies among pyside modules *might not* be the same as those among PyQt5 modules. But as I said, this will do for now.